### PR TITLE
IN-5129: Added Experience integrations to main Integrations docs page

### PR DESCRIPTION
**Pull request by adnanrahic**
Adding the same Experience integrations to the main Integrations page in the docs.

### CODE REVIEW

File: docs/integration/index.md [hunk 1]
```diff
@@ -473,6 +473,108 @@ target="_blank" rel="noopener">another AWS Lambda function</a>. If
...
+							<h5>Server-side rendered websites</h5>Full support for all types of SSR websites and applications.
+						</div>
+					</div>
+				</div>
+			</div>
+		</a>
+	</div>
+</div>
+
+All of the integrations require adding and [configuring the Experience script](../experience/getting-started). However, single-page applications require one more configuration step to register route changes.
```
Comment (otisg) [inline docs/integration/index.md]: Should we add an integration for "static websites"?

Comment (adnanrahic) [reply docs/integration/index.md]: I'd keep it here as part of the Experience integrations. What's your take on it? We can remove the SSR and Static Websites if you think they're not relevant.

Comment (otisg) [reply docs/integration/index.md]: I'm not sure what you are commenting on here.  I was asking if you think we should mention that this integrates with static websites, too, not just React, Vue.js, and friends.

Comment (adnanrahic) [reply docs/integration/index.md]: Oh! I didn't understand what you meant then. Yes, above this paragraph I've added the cards and Static Websites is among them. Check it out. :point_down: 
Selection_534

**Pull request merged**